src/AliasStorageHelper.php

```diff
@@ -186,7 +155,6 @@ public function createSiteAliases(PathAliasInterface $path, NodeInterface $node
     /** @var \Drupal\Core\Entity\EntityStorageInterface $path_storage */
     $path_storage = $this->entityTypeManager->getStorage('path_alias');
     if ($node) {
-      $this->getAliasUniquifier();
```

Comment:
I'm not sure we can actually get rid of this. I think it checks that there is not an alias that exists for other nodes (it's what appends the `_0`) to URLs when needed. Can you maybe just check with @sonnykt as he would have written this I think.

Reply:
@anthony-malkoun @vincent-gao my original code was no longer there. That service was supposed to use in `uniquify()` method but it seems that the method logic was changed.

Reply:
> I'm not sure we can actually get rid of this. I think it checks that there is not an alias that exists for other nodes (it's what appends the `_0`) to URLs when needed. Can you maybe just check with @sonnykt as he would have written this I think.

@anthony-malkoun , I think @sonnykt has confirmed that it used inside of `uniquify()` method. so I reckon it should be ok. Thoughts? or we keep it, it is harmless.